src/main/java/bt/dapps/SignumArt2.java

```diff
@@ -282,10 +285,9 @@ public void txReceived() {
 			if (getCurrentTxAmount() >= currentPrice) {
 				// Conditions match, let's execute the sale
 				pay(); // pay the current owner
+				sendMessage(getCurrentTxSender().getId(), getCurrentTxAmount(), owner.getId(),trackNewOwner);

```

Comment:
So the idea is to also send the previous owner to the tracker?

Reply:
Yes